### PR TITLE
Turn on public caching for popular HOC levels

### PR DESCRIPTION
**Pull request by philbogle**
Enables public caching for these levels:

/s/frozen/stage/1/puzzle/*
/s/gumball/stage/1/puzzle/*
/hoc/*
/s/mc/stage/1/puzzle/*
/s/starwars/stage/1/puzzle/*
/s/starwarblocks/stage/1/puzzle/*

With Josh's work and the sign-out fix we can turn this on for all of these places.

### CODE REVIEW

**Comment by philbogle** [comment]
@Bjvanminnen @breville FYI

File: cookbooks/cdo-varnish/libraries/http_cache.rb [hunk 1]
```diff
@@ -94,15 +94,17 @@ def self.config(env)
...
-          # the benefits of the CDN.
-          #
-          # {
-          # path: "/s/starwars/stage/1/puzzle/*",
-          # headers: LANGUAGE_HEADER,
-          # cookies: LANGUAGE_COOKIES
-          # },
+          {
+            path: %w{
+               /s/starwars/stage/1/puzzle/*
```
Comment (breville) [inline cookbooks/cdo-varnish/libraries/http_cache.rb:99]: can we get starwarsblocks too?  it will probably be more popular than starwars

Comment (philbogle) [inline cookbooks/cdo-varnish/libraries/http_cache.rb:99]: Great catch, added.

**Comment by ashercodeorg** [comment]
Is there a reason we can't add flappy?

**Comment by philbogle** [comment]
Just trying to keep the test matrix from getting too big.  The key to
making pages publicly cachable is eliminating dependencies on session
state.  There is a least one area (trophies) where we haven't eliminated
that dependence.  We've verified through review that none of the levels on
our whitelist use that feature, would need to verify this for flappy as
well and add it to our manual testing.  (Or decide that its OK for users
not to see trophies.)

File: cookbooks/Berksfile.lock [hunk 2]
```diff
@@ -29,6 +29,8 @@ DEPENDENCIES
     path: cdo-secrets
   cdo-solr
     path: cdo-solr
+  cdo-sqs
+    path: cdo-sqs
```
Comment (wjordan) [inline cookbooks/Berksfile.lock]: I don't see this cookbook anywhere in this PR or on staging, is this supposed to be in this PR or are these entries supposed to be removed?

**Comment by wjordan** [comment]
LGTM, just the question about the Berksfile.lock entry.

**Comment by philbogle** [comment]
Reverted the Berksfile.lock change.  This file will be checked in after Josh's changes that it depends on.

**Pull request merged**